actions/web_actions.js

```diff
@@ -149,8 +100,8 @@ export const deleteSelectedWebImages = (token, photoId) => {
         })
             .then(resp => {
                 dispatch({
-                    type: REMOVE_WEB_IMAGE,
-                    payload: photoId
+                    type: DELETE_IMAGE,
```

Comment:
remove_web_image and delete_image are different ?

Reply:
DELETE_IMAGE is the new type used in image_reducer.
it deletes the image data from state